composer.json

```diff
@@ -76,7 +76,7 @@
         "ezyang/htmlpurifier": "^4.6",
         "cebe/markdown": "~1.0.0 | ~1.1.0 | ~1.2.0",
         "bower-asset/jquery": "3.6.*@stable | 3.5.*@stable | 3.4.*@stable | 3.3.*@stable | 3.2.*@stable | 3.1.*@stable | 2.2.*@stable | 2.1.*@stable | 1.11.*@stable | 1.12.*@stable",
-        "bower-asset/inputmask": "~3.2.2 | ~3.3.5",
+        "bower-asset/inputmask": "*",
```

Comment:
[Using `*` is dangerous](https://thephp.cc/articles/the-death-star-version-constraint). It's better to add another version.

Reply:
> [Using `*` is dangerous](https://thephp.cc/articles/the-death-star-version-constraint). It's better to add another version.

Okay, but why unit tests failed on php8?

Reply:
> Okay, but why unit tests failed on php8?

Failures are not InputMask related. 
https://github.com/yiisoft/yii2/actions/runs/4041727523/jobs/6948596350#step:11:149